docs/src/main/sphinx/release/release-393.md

```diff
@@ -0,0 +1,120 @@
+# Release 393 (10 August 2022)
+
+## General
+
+## Security
+
+## Web UI
+
+## JDBC driver
+
+## Server RPM
+
+## Docker image
+
+## CLI
+
+## Accumulo connector
+
+## Atop connector
+
+## BigQuery connector
+
+## Blackhole connector
+
+## Cassandra connector
+
+## ClickHouse connector
+
+* Improve performance for queries with selective joins through push down of
+  dynamic filters to the data source. ({issue}`13334`)
+
+## Delta Lake connector
+
+## Druid connector
+
+* Improve performance for queries with selective joins through push down of
+  dynamic filters to the data source. ({issue}`13334`)
+
+## Elasticsearch connector
+
+## Google Sheets connector
+
+## Hive connector
+
+* Fix bug where query constraints were not used when fetching table statistics. ({issue}`13488`)
+
+## Iceberg connector
+
+## JMX connector
+
+## Kafka connector
+
+## Kinesis connector
+
+## Kudu connector
+
+## Local file connector
+
+## MariaDB connector
+
+* Improve performance for queries with selective joins through push down of
+  dynamic filters to the data source. ({issue}`13334`)
+
+## Memory connector
+
+## MongoDB connector
+
+## MySQL connector
+
+* Improve performance for queries with selective joins through push down of
+  dynamic filters to the data source. ({issue}`13334`)
+
+## Oracle connector
+
+* Improve performance for queries with selective joins through push down of
+  dynamic filters to the data source. ({issue}`13334`)
+
+## Phoenix connector
+
+* Improve performance for queries with selective joins through push down of
+  dynamic filters to the data source. ({issue}`13334`)
+
+## Pinot connector
+
+* Add support for the Pinot `bytes` type. ({issue}`13427`)
+
+## PostgreSQL connector
+
+* Improve performance for queries with selective joins through push down of
+  dynamic filters to the data source. ({issue}`13334`)
+* Prevent creating or renaming to a schema with a name longer than the max
```

Comment:
Just feels to long .. here is an non-optimal alternative

* Disallow long schema names during creation or renaming, instead of using shortened name siltently